tensorflow/compiler/mlir/lite/transforms/optimize.cc

```diff
@@ -206,6 +206,30 @@ DenseElementsAttr GetShape(Value output_val) {
       llvm::makeArrayRef(shape));
 }
 
+static Type getShapeStrippedType(TypeAttr type_attr)
```

Comment:
rename it to `GetShapeStrippedType` and have the "{" in the same line of the function name.